src/main/java/uk/sky/cqlmigrate/CqlPaths.java

```diff
@@ -1,19 +1,24 @@
 package uk.sky.cqlmigrate;
 
+import com.google.common.base.Throwables;
```

Comment:
`Throwables.propagate()`for handling `Files.newDirectoryStream()`'s exception within the lambda.

Same code was in the original internal (working!) implementation of cqlmigrate to propagate the checked exception.